### PR TITLE
feat(one-click-migration): add strategy options modal

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                })}
+                )
+              </StyledTitleDataValue>
+            </ContainerBox>
+          </StyledTitleDataContainer>
+          <StyledTitleDataContainer>
+            <StyledTitleData>
+              <FormattedMessage description="earn.one-click-migration.confirm-migration.apy" defaultMessage="APY" />
+            </StyledTitleData>
+            <StyledTitleDataValue>{selectedFarm.farm.apy}%</StyledTitleDataValue>
```
Comment (th0rOdinson) [inline apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx:243]: toFixed apy?

File: apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+        ))}
+      </ContainerBox>
+      <ContainerBox justifyContent="center" gap={2}>
+        <Button variant="contained" onClick={handleOnGoToStrategy} fullWidth size="large">
+          <FormattedMessage
+            description="earn.one-click-migration.confirm-migration.continue-in-vault"
+            defaultMessage="Continue in vault"
+          />
+        </Button>
+      </ContainerBox>
```
Comment (th0rOdinson) [inline apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx:279]: Should add the Go Back button too

File: apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+                      <FeeItem fee={fee} intl={intl} />
+                    </Grid>
+                  ))}
+                </Grid>
+              }
+            >
+              <ContainerBox>
+                <InfoCircleIcon fontSize="small" />
+              </ContainerBox>
+            </Tooltip>
```
Comment (th0rOdinson) [inline apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx:135]: Is this 'promoted' style necessary for the release?
Screenshot 2024-12-04 at 10 22 51 AM

File: apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx (hunk 2 above)
Comment (FiboApe) [reply apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx:279]: Actually no, we have it on the top :)

File: apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx (hunk 3 above)
Comment (FiboApe) [reply apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx:135]: Yes and I missed it!

**Pull request merged**